### PR TITLE
Update contacts

### PR DESCRIPTION
**Pull request by ghiobi**
users are now allowed to update contact information.

### CODE REVIEW

File: app/Http/Controllers/User/ContactController.php [hunk 1]
```diff
@@ -85,10 +85,22 @@ public function store(Request $request)
      * @param  int  $id
      * @return \Illuminate\Http\Response
      */
-//    public function update(Request $request, $id)
-//    {
-//        //
-//    }
+    public function update(Request $request, $id)
```
Comment (ajpower) [inline app/Http/Controllers/User/ContactController.php]: Good job removing the function skeleton.

File: tests/feature/ContactTest.php [hunk 2]
```diff
@@ -51,6 +51,8 @@ public function testStore()
 
     /**
      * Destroy contact test.
+     *
+     * @return void
```
Comment (ajpower) [inline tests/feature/ContactTest.php]: Is it necessary to specify return void?

Comment (ghiobi) [inline tests/feature/ContactTest.php]: Yes, all my other tests have return void. Lets' keep it consistent.

Comment (JTalarico) [inline tests/feature/ContactTest.php]: adb shell am start -a android.intent.action.VIEW -d "ionichats://app/hats/very-nice-hat" io.ionic.hats

**Pull request merged**